composer.lock

```diff
@@ -1896,16 +1897,16 @@
         },
         {
             "name": "symfony/string",
-            "version": "v5.2.1",
+            "version": "v5.2.3",
```

Comment:
I was trying to figure out why dependabot isn't increasing these. I think it comes down to `version_strategy` https://docs.github.com/en/github/administering-a-repository/configuration-options-for-dependency-updates#versioning-strategy